ocaml/xenopsd/lib/xenops_server.ml

```diff
@@ -2731,10 +2727,9 @@ and perform_exn ?subtask ?result (op : operation) (t : Xenops_task.task_handle)
                    the destination host. even though the destination host
                    failed to respond successfully to our handshake, the VM
                    should still be running correctly *)
-                error
+                internal_error
```

Comment:
This changes the string of the exception compared to the old code. But it just adds more info to the message, which should be fine.